src/pages/Graph/GraphHelpFind.tsx

```diff
@@ -185,6 +185,11 @@ export default class GraphHelpFind extends React.Component<GraphHelpFindProps> {
 
   render() {
     const className = this.props.className ? this.props.className : '';
+    const preface =
+      'The Find and Hide input boxes are used to highlight or remove graph nodes and edges. Each accepts ' +
```

Comment:
"remove" might sound scary :)
perhaps hide/not display?

Reply:
You're right, hide is better.

Reply:
+1